applications/plugins/SofaSphFluid/src/SofaSphFluid/OglFluidModel.inl

```diff
@@ -368,7 +368,10 @@ void OglFluidModel<DataTypes>::drawSprites(const core::visual::VisualParams* vpa
 
     Mat4x4f matProj(fProjMat);
     Mat4x4f invmatProj;
-    invmatProj.invert(matProj);
+    if(!invmatProj.invert(matProj))
+    {
+        msg_error() << "matProj can not be inverted";
```

Comment:
```suggestion
        msg_error() << "matProj can not be inverted";
```
I suggest a message error more designed to the user

Reply:
@hugtalbot choose the easy path. 
More seriously, if the inversibility depends on user data (scene) and thus can be fixed by changing something in the scene then it is not ok to use a dmsg.